bundles/org.openhab.binding.pentair/README.md

```diff
@@ -72,133 +87,200 @@ The following table shows the available configuration parameters for each thing.
+
+### Thing: IntelliChlor
+
+Represents an Intellichlor module connected in your system.  Currently, the values here are readonly.
+
+| Channel              | Type       |     | Description |
+| :------------------: | :----:     | :-: | :---------- |
+| saltoutput           | Number     | R   | Current salt output %. |
+| salinity             | Number     | R   | Salinity (ppm). |
+| ok                   | Switch     | R   | System is operating normally. |
+| lowflow              | Switch     | R   | Water flow rate is low. |
+| lowsalt              | Switch     | R   | Low salt level. |
+| verylowsalt          | Switch     | R   | Very low salt level. |
+| highcurrent          | Switch     | R   | High current level. |
+| cleancell            | Switch     | R   | Clean cell. |
+| lowvoltage           | Switch     | R   | Low voltage. |
+| lowwatertemp         | Switch     | R   | Water temperature is too low for chlorine generation. |
+| commerror            | Switch     | R   | Communication error. |
+
+### Thing: IntelliFlo
+
+Represents and interfaces to an Intelliflo pump.  When a controller is active in the system all pump values are read only since the pump can only have one master at a time.  If no controller is present or the controller is in service mode, the pump can be controlled directly from OpenHab.
+
+| Channel              | Type       |     | Description |
+| :------------------: | :----:     | :-: | :---------- |
+| run                  | Switch    | RW | Indicates whether the pump is running. |
+| rpm                  | Number    | RW  | Pump RPM |
+| gpm                  | Number    | R  | Pump GPM (only valid for VF pumps) |
+| power                | Number:Power   | R  | Pump power (Watt) |
+| status1              | Number    | R  | Pump status1. |
+| status2              | Number    | R  | Pump status2. |
+| program1             | Switch    | RW | Run pump program 1 settings. |
+| program2             | Switch    | RW | Run pump program 2 settings. |
+| program3             | Switch    | RW | Run pump program 3 settings. |
+| program4             | Switch    | RW | Run pump program 4 settings. |
+
+
+### Thing: IntelliChem
+
+Represents and interfaces to an IntelliChem unit. This is for montioring of values only and IntelliChem cannot be directly controlled through this binding.
+
+| Channel              | Type       |     | Description |
+| :------------------: | :----:     | :-: | :---------- |
+| phreading            | Number     | R   | Current PH reading. |
+| orpreading           | Number     | R   | Current Oxidation Reduction Potential (ORP) reading. |
+| phsetpoint           | Number     | R   | Current PH set point. |
+| orpsetpoint          | Number     | R   | Oxidation Reduction Potential (ORP) set point. |
+| tank1                | Number     | R   | Tank 1 level. |
+| tank2                | Number     | R   | Tank 2 level. |
+| calciumhardness      | Number     | R   | Calcium hardness PPM (mg/L). |
```

Comment:
Strictly speaking it can either be ppm or mg/L. Doesn't make much difference in practice, probably. So, you could make this `Number:Dimensionless`.